packages/ag-branding/src/Logo.stories.tsx

```diff
@@ -1,22 +1,14 @@
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 import { Box } from '@ag.ds-next/box';
-import { Logo } from './Logo';
+import { Logo as AgLogo } from './Logo';
 
 export default {
 	title: 'Brand/AG Branding/Logo',
-	component: Logo,
-} as ComponentMeta<typeof Logo>;
+	component: AgLogo,
+} as ComponentMeta<typeof AgLogo>;
 
-export const OnLight: ComponentStory<typeof Logo> = () => (
+export const Logo: ComponentStory<typeof AgLogo> = () => (
```

Comment:
This story no longer works in dark mode as the fill of the icon of black and the background colour is dark. 

Adding `color="text"` to the `Box` in `withBrandTheme` fixes this issue.